### PR TITLE
Translate license.po

### PR DESCRIPTION
**Pull request by StevenHsuYL**
Complete translation of license.po
Thanks for review.

Reference: https://terms.naer.edu.tw/

### CODE REVIEW

File: license.po [hunk 1]
```diff
@@ -34,13 +36,18 @@ msgid ""
...
+"Guido van Rossum 於 1990 年代早期所創造，目的是作為一種稱為 ABC 語言的後繼"
+"者。儘管 Python 包含了許多來自其他人的貢獻，Guido 仍是其主要作者。"
 
 #: ../../license.rst:18
 msgid ""
 "In 1995, Guido continued his work on Python at the Corporation for National "
 "Research Initiatives (CNRI, see https://www.cnri.reston.va.us/) in Reston, "
 "Virginia where he released several versions of the software."
 msgstr ""
+"1995 年，Guido在維吉尼亞州雷斯頓的國家創新研究公司（CNRI，見 https://www."
```
Comment (mattwang44) [inline license.po]:
```suggestion
"1995 年，Guido 在維吉尼亞州雷斯頓的國家創新研究公司（CNRI，見 https://www."
```

File: license.po [hunk 2]
```diff
@@ -52,6 +59,11 @@ msgid ""
 "specifically to own Python-related Intellectual Property.  Zope Corporation "
 "is a sponsoring member of the PSF."
 msgstr ""
+"2000 年五月，Guido 和 Python 核心開發團隊轉移到 BeOpen.com 並成立了 BeOpen "
+"PythonLabs 團隊。同年十月，PythonLabs 團隊轉移到 Digital Creations（現為 "
+"Zope Corporation；見 https://www.zope.org/）。2001年，Python 軟體基金會"
```
Comment (mattwang44) [inline license.po]:
```suggestion
"Zope Corporation；見 https://www.zope.org/）。2001 年，Python 軟體基金會"
```

File: license.po [hunk 3]
```diff
@@ -206,16 +221,45 @@ msgid ""
...
 msgstr ""
+"Python 軟體和說明文件的授權是基於 :ref:`PSF 授權合約 <PSF-license>`\\ 。"
+
+#: ../../license.rst:75
+msgid ""
+"Starting with Python 3.8.6, examples, recipes, and other code in the "
+"documentation are dual licensed under the PSF License Agreement and the :ref:"
+"`Zero-Clause BSD license <BSD0>`."
+msgstr ""
+"從 Python 3.8.6開始，說明文件中的範例、程式庫和其他程式碼，是被雙重授權 "
```
Comment (mattwang44) [inline license.po]:
```suggestion
"從 Python 3.8.6 開始，說明文件中的範例、程式庫和其他程式碼，是被雙重授權 "
```

File: license.po [hunk 4]
```diff
@@ -239,134 +283,174 @@ msgstr ""
...
-msgstr ""
+msgstr "被收錄軟體的授權與致謝"
 
 #: ../../license.rst:298
 msgid ""
 "This section is an incomplete, but growing list of licenses and "
 "acknowledgements for third-party software incorporated in the Python "
 "distribution."
 msgstr ""
+"本節是一個不完整且不斷成長的授權與致謝清單，對象是在 Python 發佈版本中所收錄"
```
Comment (mattwang44) [inline license.po]:
```suggestion
"本節是一個不完整但持續增加的授權與致謝清單，對象是在 Python 發佈版本中所收錄"
```

File: license.po [hunk 5]
```diff
@@ -239,134 +283,174 @@ msgstr ""
...
+msgstr "Sockets"
 
 #: ../../license.rst:354
 msgid ""
 "The :mod:`socket` module uses the functions, :func:`getaddrinfo`, and :func:"
 "`getnameinfo`, which are coded in separate source files from the WIDE "
 "Project, http://www.wide.ad.jp/. ::"
 msgstr ""
+":mod:`socket` 模組使用 :func:`getaddrinfo` 和 :func:`getnameinfo` 函式，它們"
+"在 WIDE 專案（http://www.wide.ad.jp/）內，於不同的原始檔案中被編碼。\n"
```
Comment (mattwang44) [inline license.po]:
```suggestion
"在 WIDE 專案（http://www.wide.ad.jp/）內，於不同的原始檔案中被編碼：\n"
```

**Comment by mattwang44** [comment]
@StevenHsuYL I noticed that several new source strings are added after this PR is opened. Would you mind also translating them?

**Comment by StevenHsuYL** [comment]
> noticed that several new source strings are added after this PR is opened. Would you mind also translating them?

Of course not! Could you point out the newly added source strings? 
BTW, I have also updated the "masOS" term here, which was updated in cpython on Sep, 23 by [GH-28515](https://github.com/python/cpython/pull/28524).

**Comment by mattwang44** [comment]
> Of course! Could you point out the newly added source strings?

You can refer to [this link](https://github.com/python/python-docs-zh-tw/blob/5373f0bd71cfeb3085b4bba81aec8098758f7574/license.po#L264-L282) for the untranslated source strings in `license.po`. 

> BTW, I have also updated the "masOS" term here, which was updated on Sep, 23 by GH-28515.

Yeah I've noticed that you discover the modification in CPython and made the change in advance! That's pretty nice! I just want to mention that we have a [GitHub Action workflow](https://github.com/python/python-docs-zh-tw/blob/3.9/.github/workflows/py39-sync-cpython.yml) checks if any string in CPython is modified and sends a PR if so (such as [this PR](https://github.com/python/python-docs-zh-tw/pull/112) also handles the changes of `macOS` terms). So we probably do not need to spend effort on checking CPython doc from now on!

**Comment by StevenHsuYL** [comment]
> > Of course! Could you point out the newly added source strings?
> 
> You can refer to [this link](https://github.com/python/python-docs-zh-tw/blob/5373f0bd71cfeb3085b4bba81aec8098758f7574/license.po#L264-L282) for the untranslated source strings in `license.po`.

I did. I found these three "untranslated" sentences had already been translated in the [above codes](https://github.com/python/python-docs-zh-tw/blob/5373f0bd71cfeb3085b4bba81aec8098758f7574/license.po#L239-L262), 
Therefore, they're "redundant" source strings. 

> > BTW, I have also updated the "masOS" term here, which was updated on Sep, 23 by GH-28515.
> 
> Yeah I've noticed that you discover the modification in CPython and made the change in advance! That's pretty nice! I just want to mention that we have a [GitHub Action workflow](https://github.com/python/python-docs-zh-tw/blob/3.9/.github/workflows/py39-sync-cpython.yml) checks if any string in CPython is modified and sends a PR if so (such as [this PR](https://github.com/python/python-docs-zh-tw/pull/112) also handles the changes of `macOS` terms). So we probably do not need to spend effort on checking CPython doc from now on!

Great! It's excellent that we can sync with CPython automatically!

**Comment by mattwang44** [comment]
aw, i got it!
I'll take those string down if needed. Thanks again for your effort!

**Review by mattwang44 — APPROVED** [review]
LGTM

**Pull request merged**